app/views/profiles/_form.html.erb

```diff
@@ -18,6 +18,39 @@
                    label: "Nickname",
                    help: "Unique. Reserved for future use on menadevs.com" %>
 
+
+
+<label>Skills</label>
+<select id="skill_list" name="skill_list[]" class="skills-input" class="js-example-basic-single" multiple="multiple">
+
+  User.where(weekly_subscriber: true).find_each do |user|
+    NewsMailer.weekly(user).deliver_now
+  end
+
+
+  <% Tag.where(is_approved: true).find_each do |skill| %>
+    <option value="<%= skill.name %>"><%= skill.name %></option>
+  <% end %>
+</select>
+<span class="help-block skills-help">Note: if a skill doesn't exist in the list, manually type it and press enter.</span>
+
+<script type="text/javascript">
```

Comment:
It is cleaner to move it, but there are 2 problems:

1- This code is only needed in this specific form
2- The script actually depends on this ruby variable: **@profile.skill_list**

So maybe I can move it to the end of the file as you suggested? unless you have another solution in mind